File: Python/EstruturaDeDecisao/exercicio_25.py
"""
Faça um programa que faça 5 perguntas para uma pessoa sobre um crime.
As perguntas são:

1. "Telefonou para a vítima?"
2. "Esteve no local do crime?"
3. "Mora perto da vítima?"
4. "Devia para a vítima?"
5. "Já trabalhou com a vítima?"

O programa deve no final emitir uma classificação sobre a participação da pessoa no crime.
Se a pessoa responder positivamente a 2 questões ela deve ser classificada como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino".
Caso contrário, ele será classificado como "Inocente".
"""

perguntas = [
    "Telefonou para a vítima?",
    "Esteve no local do crime?",
    "Mora perto da vítima?",
    "Devia para a vítima?",
    "Já trabalhou com a vítima?"]

print("Responda 'sim' ou 'não' para as perguntas abaixo:")

contador = 0
for pergunta in perguntas:
    resposta = input(pergunta + " (sim ou não): ").lower()
    while resposta not in ['sim', 'não']:
        print("Resposta inválida. Responda apenas com 'sim' ou 'não'.")
        resposta = input(pergunta + " (sim ou não): ").lower()
    if resposta == 'sim':
        contador += 1

if contador == 2:
    print("Classificação: Suspeita")
elif 3 <= contador <= 4:
    print("Classificação: Cúmplice")
elif contador == 5:
    print("Classificação: Assassino")
else:
    print("Classificação: Inocente")

File: Java/EstruturaDeDecisao/Exercicio25.java
// Faça um programa que faça 5 perguntas para uma pessoa sobre um crime.
// As perguntas são:
// - "Telefonou para a vítima?"
// - "Esteve no local do crime?"
// - "Mora perto da vítima?"
// - "Devia para a vítima?"
// - "Já trabalhou com a vítima?"

// O programa deve no final emitir uma classificação sobre a participação da pessoa no crime.
// Se a pessoa responder positivamente a 2 questões ela deve ser classificada como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino".
// Caso contrário, ele será classificado como "Inocente".
package EstruturaDeDecisao;
import java.util.Scanner;

public class Exercicio25 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Responda SIM ou NÃO para as perguntas a seguir:");

        int pontos = 0;

        System.out.print("Telefonou para a vítima? ");
        String resposta1 = scanner.next().toUpperCase();
        if (resposta1.equals("SIM")) {
            pontos++;
        }

        System.out.print("Esteve no local do crime? ");
        String resposta2 = scanner.next().toUpperCase();
        if (resposta2.equals("SIM")) {
            pontos++;
        }

        System.out.print("Mora perto da vítima? ");
        String resposta3 = scanner.next().toUpperCase();
        if (resposta3.equals("SIM")) {
            pontos++;
        }

        System.out.print("Devia para a vítima? ");
        String resposta4 = scanner.next().toUpperCase();
        if (resposta4.equals("SIM")) {
            pontos++;
        }

        System.out.print("Já trabalhou com a vítima? ");
        String resposta5 = scanner.next().toUpperCase();
        if (resposta5.equals("SIM")) {
            pontos++;
        }

        String classificacao = "";
        if (pontos == 2) {
            classificacao = "Suspeita";
        } else if (pontos >= 3 && pontos <= 4) {
            classificacao = "Cúmplice";
        } else if (pontos == 5) {
            classificacao = "Assassino";
        } else {
            classificacao = "Inocente";
        }

        System.out.println("Classificação: " + classificacao);

        scanner.close();
    }
}
